src/Components/Articles.jsx

```diff
@@ -32,10 +44,15 @@ function Articles() {
         {article.body}
         <br />
         <br />
-        Comments: {article.comment_count}
+        <div
```

Comment:
Why is this a div and not a button? It is not clear to even a fully sighted person that this is something I can click on. A screen reader will completely ignore this and not know. I also appreciate that a button shouldn't be used for navigation. I would argue that a user should not be navigated to another page to view the comments. If you want to hide and show the comments to make the page tidier consider conditionally rendering the comments component on the same page.